languages/elixir/exercises/concept/basic-strings/.docs/hints.md

```diff
@@ -0,0 +1,32 @@
+### General
+
+- Read about strings in the official [Getting Started Elixir guide][getting-started-strings].
+- Browse the [functions available in the _String module_][string-module-functions] to discover which operations on strings Elixir's standard library offers.
+
+### 1. Get the name's first letter
+
+- Use [`String.first/1`][string-first] to get the first character from a string.
```

Comment:
The hints in this file give away the solution. Quoting the [hints.md documentation](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/strings/.docs/hints.md):

> The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).

Maybe you could phrase things such that the solution is not given away directly, but that you _do_ still point to the right resource? For an example, see [the hints.md file for F#'s strings exercise](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/strings/.docs/hints.md).

Reply:
I rewrote the hints to avoid commands ("use this and that"). To be less obvious, I added links to alternative functions where either or the one fits (e.g. `String.first(s)` vs `String.at(s, 0)`), but also to some related functions that don't need to be used (`String.last`).